### ISSUE TITLE
WebSocket does not connect

### ISSUE THREAD

**Issue by samucs** [vf1/sipserver#2]
Hello, I am trying OfficeSIP 3.5 and SIP-JS in order to perform video calls from my Chrome browser. 
However, WebSocket fails to connect.

``` javascript
createSocket() transport=ws
    connecting to localhost:5060
socket connected=false
```

Then Chrome displays the message:

``` javascript
WebSocket connection to 'ws://localhost:5060/sip' failed: Connection closed before receiving a handshake response
```

Anybody has the solution for this problem? Is there any WebSockets configuration?
Thanks in advance!

**Comment by tilleul** [comment]
I have noticed the same problem using sipJS or JSsip... interested in any solution ...

**Comment by justindstein** [comment]
Also seeing this issue in jssip w/ chrome 66. The issue seems to resolve itself if you clear all cookies from your browser, but I haven't figured out how/why. Did @tilleul or @samucs ever figure this one out?

**Comment by tilleul** [comment]
Nope. I haven't tried since :) ... So you're saying it's working with Chrome 66 ?

**Comment by justindstein** [comment]
Not sure about this particular project. But in the project I'm working on, this error occurs *sometimes*. Just stumbled upon this thread on google!
